src/applications/personalization/connected-accounts/reducers/connectedAccounts.js

```diff
@@ -0,0 +1,63 @@
+import {
```

Comment:
this was all pulled out of the main reducer in `connected-accounts` so that it can also be used in the main `account` reducer.